<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmTiledBg</class>
 <widget class="QWidget" name="frmTiledBg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="TiledBg" name="tiledBg1" native="true"/>
   </item>
   <item row="0" column="1">
    <widget class="TiledBg" name="tiledBg2" native="true"/>
   </item>
   <item row="1" column="0">
    <widget class="TiledBg" name="tiledBg3" native="true"/>
   </item>
   <item row="1" column="1">
    <widget class="TiledBg" name="tiledBg4" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TiledBg</class>
   <extends>QWidget</extends>
   <header>tiledbg.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
